README.md

```diff
@@ -630,6 +650,11 @@ deduction than the prosecutor's proposal and should be a reduction of at least 0
 	point reduction. The rounded average (rounded to one decimal place) of these votes will be the final verdict of the
 	jury.
 
+**378** *The Defense Culpability Act*
+
+If the defendant is found _Guilty_ and was represented by a defense attorney at the time jury deliberation started, this
```

Comment:
"and there was no "self-representation". (Technically one could argue a defendent is representing themselves as an attorney and hence get double punishment)